ports/windows/gccollect.c

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
Same remark as the other file: I'm not sure there even is a functional difference with unix/gccollect.c?

Reply:
There's windows api used. I'm not sure it's ok to put them in unix folder. So I made a copy.

Reply:
If it's only a slight change, or a function which has the same signature in POSIX but just a different name, it's ok to do this I think. Else the way this has be fixed until now is to define our own implementation of the posix function (for example like is done for realpath/gettimeofday/opendir/...).